Bottom: a40603939d80c56d51154f76cac45f787f28fec1
Top:    d5099c914091430c11100e08af89a30fd3df9720
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-10 03:40:38 +0000

Refresh of don-t-make-gui-drawing-calls-t

---

diff --git a/src/autoroute.c b/src/autoroute.c
index 1b8b82f..e53587c 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -103,7 +103,7 @@ RCSID ("$Id$");
 //#define DEBUG_SHOW_ZIGZAG
 */
 
-static hidGC ar_gc = 0;
+//static hidGC ar_gc = 0;
 
 #define EXPENSIVE 3e28
 /* round up "half" thicknesses */
